src/prefect/deployments.py

```diff
@@ -72,6 +72,8 @@ async def run_deployment(
             Setting `timeout` to None will allow this function to poll indefinitely.
             Defaults to None
         poll_interval: The number of seconds between polls
+        tags: An optional list of tags to associate with this flow run; note that tags are used only for organizational purposes.
```

Comment:
You can remove 'optional' here.. as many of these params are, without explicitly saying so.

Also consider adding in this PR, `Optional[type]` tags above to each of the items with defaults (Everything but `name`). but this style should be something the Prefect team should weigh in on. It's _nice_ but also clutters readability.